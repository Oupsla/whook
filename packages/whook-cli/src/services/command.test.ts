import initCommand from './command';

describe('command', () => {
  const log = jest.fn();
  const commandHandler = () => log('commandHandler ran');

  beforeEach(() => {
    log.mockReset();
  });

  it('should work', async () => {
    const command = await initCommand({
      commandHandler,
      log,
    });

    command();

    expect({
      logCalls: log.mock.calls.filter(args => 'stack' !== args[0]),
    }).toMatchSnapshot();
  });
});
