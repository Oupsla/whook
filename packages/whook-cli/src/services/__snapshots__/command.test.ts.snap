// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`command should work 1`] = `
Object {
  "logCalls": Array [
    Array [
      "commandHandler ran",
    ],
  ],
}
`;
